### PR TITLE
Get endpoint with data

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: core/app_feeds.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import re
+
+import aiohttp
+import mohawk
+
+from .app_utils import (
```
Comment (lgarvey) [inline core/app_feeds.py]: Do you need the () here?

File: core/app_feeds.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    sub_dict_lower,
+)
+
+
+class ElasticsearchBulkFeed():
```
Comment (lgarvey) [inline core/app_feeds.py]: () not needed ?

File: core/app_feeds.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        return cls(**sub_dict_lower(config, ['SEED', 'ACCESS_KEY_ID', 'SECRET_ACCESS_KEY']))
+
+    def __init__(self, seed, access_key_id, secret_access_key):
+        self.seed = seed
+        self.access_key_id = access_key_id
+        self.secret_access_key = secret_access_key
+
+    @staticmethod
+    def next_href(feed):
+        return feed['next_url'] if 'next_url' in feed else None
```
Comment (lgarvey) [inline core/app_feeds.py]: return feed.get('next_url', None)   ?

File: core/app_feeds.py (hunk 1 above)
Comment (michalc) [reply core/app_feeds.py]: Removed!

File: core/app_feeds.py (hunk 2 above)
Comment (michalc) [reply core/app_feeds.py]: Removed!

File: core/app_feeds.py (hunk 3 above)
Comment (michalc) [reply core/app_feeds.py]: Changed!

**Pull request merged**